package/CHANGELOG

```diff
@@ -48,6 +49,7 @@ Fixes
   * Added parmed to setup.py
 
 Enhancements
+  * encore.hes() always returns details, consistently with ces and dres (Issue #2465)
```

Comment:
Are these really two different things that the PR does? To me it looks like a single entry under *Changes*.